lib/Twig/Node/Expression/Array.php

```diff
@@ -8,74 +8,140 @@
+    protected $toArray = array ();
+
+    public function __construct ( array $elements , $lineno )
     {
-        parent::__construct($elements, array(), $lineno);
+        parent::__construct ( $elements , array () , $lineno );
 
-        $this->index = -1;
-        foreach ($this->getKeyValuePairs() as $pair) {
-            if ($pair['key'] instanceof Twig_Node_Expression_Constant && ctype_digit((string) $pair['key']->getAttribute('value')) && $pair['key']->getAttribute('value') > $this->index) {
-                $this->index = $pair['key']->getAttribute('value');
+        $this->index = - 1;
+        foreach ( $this->getKeyValuePairs () as $pair )
+        {
+            if ( $pair[ 'key' ] instanceof Twig_Node_Expression_Constant
+                 && ctype_digit ( (string) $pair[ 'key' ]->getAttribute ( 'value' ) )
+                 && $pair[ 'key' ]->getAttribute ( 'value' ) > $this->index
+            )
+            {
+                $this->index = $pair[ 'key' ]->getAttribute ( 'value' );
             }
         }
     }
 
-    public function getKeyValuePairs()
+    public function getKeyValuePairs ()
     {
-        $pairs = array();
+        $pairs = array ();
 
-        foreach (array_chunk($this->nodes, 2) as $pair) {
-            $pairs[] = array(
-                'key' => $pair[0],
-                'value' => $pair[1],
+        foreach ( array_chunk ( $this->nodes , 2 ) as $pair )
+        {
+            $pairs[] = array (
+                'key'   => $pair[ 0 ] ,
+                'value' => $pair[ 1 ] ,
             );
         }
 
         return $pairs;
     }
 
-    public function hasElement(Twig_Node_Expression $key)
+    public function hasElement ( Twig_Node_Expression $key )
     {
-        foreach ($this->getKeyValuePairs() as $pair) {
+        foreach ( $this->getKeyValuePairs () as $pair )
+        {
             // we compare the string representation of the keys
             // to avoid comparing the line numbers which are not relevant here.
-            if ((string) $key === (string) $pair['key']) {
+            if ( (string) $key === (string) $pair[ 'key' ] )
+            {
                 return true;
             }
         }
 
         return false;
     }
 
-    public function addElement(Twig_Node_Expression $value, Twig_Node_Expression $key = null)
+    public function addElement ( Twig_Node_Expression $value , Twig_Node_Expression $key = null )
     {
-        if (null === $key) {
-            $key = new Twig_Node_Expression_Constant(++$this->index, $value->getTemplateLine());
+        if ( null === $key )
+        {
+            $key = new Twig_Node_Expression_Constant( ++ $this->index , $value->getTemplateLine () );
         }
 
-        array_push($this->nodes, $key, $value);
+        array_push ( $this->nodes , $key , $value );
     }
 
-    public function compile(Twig_Compiler $compiler)
+    public function compile ( Twig_Compiler $compiler )
```

Comment:
why are you changing coding standards ? If you want it to be merged, this must be reverted (and if you don't want it to be merged, you should close the PR)